server/keycloak.js

```diff
@@ -26,16 +26,16 @@ const defaults = {
 // However, it can be used once we switch to TS.
 /**
  * @typedef {Object} hcapUserInfo
- * @property {string} name           Full name of user (e.g. "Tabitha Test")
- * @property {string} username       Username (e.g. "user@bceid")
- * @property {string} id             Unique identifier string for the user
- * @property {any[]} sites TODO: annotate element type
- * @property {string[]} roles        Roles of the user (such as `region_interior` or `health_authority`)
- * @property {string[]} regions      Regions the user is assigned to (such as `Interior`)
- * @property {boolean} isEmployer    True if the user is an employer
- * @property {boolean} isHA          True if the user is a health authority user
- * @property {boolean} isSuperUser   True if the user is a superuser
- * @property {boolean} isMoH         True if the user is an MoH user
+ * @property {string=} name           Full name of user (e.g. "Tabitha Test")
+ * @property {string=} username       Username (e.g. "user@bceid")
```

Comment:
What is the purpose of the =?

Reply:
`=` indicates a property is optional and defaults to null.